api/db/changelog/db.changelog-4.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="blrubenstein" id="changelog-4">
+    <addColumn tableName="workspace">
+      <column name="disease_focused_research" type="boolean"/>
+      <column name="disease_of_focus" type="varchar(80)"/>
```

Comment:
Can you add a comment explaining why X characters is plenty in each case?

Reply:
I'd just bump up column sizes. Increase to 255?

Reply:
This is a single line text field. Technically users can put in a lot here, but I think that 80 will cover anything a user would want to put there. (We can do some client side filtering)

The thirty character column one is going to be a dropdown of population options, and I really don't see any option being more than 30 characters long.